models/checklist.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose'
+const Schema = mongoose.Schema
+
+
+const checklistSchema = new mongoose.Schema({
+    owner: {type: Schema.Types.ObjectId, 'ref': "Profile"},
```

Comment:
@yoni1994 Can you correct this `ref` also (remove the quotes and submit a new pull request please)